src/index.html

```diff
@@ -17,6 +17,54 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <a
+        href="#"
```

Comment:
The `href` attribute for the logo link is set to `#`, which means it doesn't lead anywhere. Consider providing a meaningful URL or using `javascript:void(0);` if the link is not intended to navigate.